src/keri/db/basing.py

```diff
@@ -1335,7 +1337,7 @@ def reload(self):
         for keys in removes:  # remove bare .habs records
             self.habs.rem(keys=keys)
 
-    def migrate(self):
+    def migrate(self, name, base, temp):
```

Comment:
These three parameters (name, base, temp) are available from the database object itself.  Please remove them and use them from `self`

Reply:
Those three arguments are now removed.